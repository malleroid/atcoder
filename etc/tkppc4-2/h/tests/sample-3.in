10 5
20 3 9 5 8 1 100 32 7 2
