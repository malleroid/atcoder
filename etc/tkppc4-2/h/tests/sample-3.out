5.723472222222
